### PR TITLE
[OSDOCS-12211]: Update Compliance Table to show ROSA with HCP is HIPAA Qualified

### PR DESCRIPTION
**Pull request by michaelryanmcneill**
Version(s): `enterprise-4.17`+

Issue: https://issues.redhat.com/browse/OSDOCS-12211

Link to docs preview: https://82878--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.html

QE review:
- [x] ~~QE has approved this change.~~ QE review is not required for this change.

Additional information: This is just updating the documentation to reflect the actual state of the product here: https://access.redhat.com/articles/compliance_activities_and_gov_standards#hipaa-overview-13

### CODE REVIEW

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

File: modules/rosa-policy-security-regulation-compliance.adoc [hunk 1]
```diff
@@ -71,7 +71,8 @@ Any issues that may be discovered are prioritized based on severity. Any issues
 
 | SOC 3 | Yes | Yes
 
-| FedRAMP High^[1]^ | Yes (link:https://aws.amazon.com/govcloud-us/?whats-new-ess.sort-by=item.additionalFields.postDateTime&whats-new-ess.sort-order=desc[GovCloud] requisite) | No
+| FedRAMP High^[2]^ | Yes (link:https://aws.amazon.com/govcloud-us/?whats-new-ess.sort-by=item.additionalFields.postDateTime&whats-new-ess.sort-order=desc[GovCloud] requisite) | No
 
 |===
-1. For more information about ROSA on GovCloud, see the link:https://marketplace.fedramp.gov/products/FR2102031769[FedRAMP Marketplace ROSA Agency] and link:https://marketplace.fedramp.gov/products/FR2102031769A[ROSA JAB listings].
\ No newline at end of file
+1. For more information about Red Hat's HIPAA Qualified ROSA offerings, see the link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#hipaa-overview-13[Red Hat Compliance Activities and Government Standards page].
```
Comment (mletalie) [inline modules/rosa-policy-security-regulation-compliance.adoc:77]:
```suggestion
1. For more information about Red Hat's HIPAA Qualified ROSA offerings, see  link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#hipaa-overview-13[HIPAA Overview].
```

File: modules/rosa-policy-security-regulation-compliance.adoc [hunk 2]
```diff
@@ -71,7 +71,8 @@ Any issues that may be discovered are prioritized based on severity. Any issues
...
+2. For more information about ROSA on GovCloud, see the link:https://marketplace.fedramp.gov/products/FR2102031769[FedRAMP Marketplace ROSA Agency] and link:https://marketplace.fedramp.gov/products/FR2102031769A[ROSA JAB listings].
```
Comment (mletalie) [inline modules/rosa-policy-security-regulation-compliance.adoc:78]:
```suggestion
2. For more information about ROSA on GovCloud, see link:https://marketplace.fedramp.gov/products/FR2102031769[FedRAMP Marketplace ROSA Agency] and link:https://marketplace.fedramp.gov/products/FR2102031769A[ROSA JAB listings].
```

**Comment by mletalie** [comment]
Hello @michaelryanmcneill,
   Couple of suggestions, otherwise /lgtm!
Will send to Eric P for merge review.

**Review by mletalie — COMMENTED** [review]
/label peer-review-done
/remove-label peer-review-in-progress

File: modules/rosa-policy-security-regulation-compliance.adoc (hunk 1 above)
Comment (michaelryanmcneill) [reply modules/rosa-policy-security-regulation-compliance.adoc]: Per discussion, kept the "the" but changed the link name as requested.

File: modules/rosa-policy-security-regulation-compliance.adoc (hunk 2 above)
Comment (michaelryanmcneill) [reply modules/rosa-policy-security-regulation-compliance.adoc:78]: Per discussion, kept the "the".

**Comment by michaelryanmcneill** [comment]
Changes made, thanks @mletalie!

**Pull request merged**